packages/orbit-components/transforms/Modal-size.js

```diff
@@ -0,0 +1,23 @@
+// @noflow
+/* eslint-disable no-param-reassign */
+function transformModalSizeProp(fileInfo, api) {
+  const j = api.jscodeshift;
+  const root = j(fileInfo.source);
+
+  return root
+    .find(j.JSXOpeningElement, { name: { name: "Modal" } })
+    .find(j.JSXAttribute, { name: { name: "size" } })
+    .forEach(path => {
+      if (path.node.value.type === j.Literal.name) {
+        path.node.value.value = "extraLarge";
+      } else {
+        console.warn(
+          "Unable to transform Modal size value because it's not a string literal, please verify whether large size is being used and use extraLarge instead.",
+          fileInfo.path,
+        );
```

Comment:
Almost, this is what I meant:
```suggestion
        const { loc } = path.node.value;
        console.warn(
          "Unable to transform Modal size value because it's not a string literal, please verify whether large size is being used and use extraLarge instead.",
          `${fileInfo.path}:${loc.start.line}:${loc.start.column}`,
        );
```
Considering that we have location data available, why not help people as much as possible so people can just straight to the location of the `size` value. 😉

Reply:
oh, yes, thanks a lot